modules/Sources/Features/ImportWallet/ImportWallet.swift

```diff
@@ -126,27 +132,13 @@ extension AlertState where Action == ImportWallet.Action.Alert {
 }
 
 extension ViewStoreOf<ImportWallet> {
-    func bindingForRedactableSeedPhrase(_ importedSeedPhrase: RedactableString) -> Binding<String> {
-        self.binding(
-            get: { _ in importedSeedPhrase.data },
-            send: { .seedPhraseInputChanged($0.redacted) }
-        )
-    }
-    
-    func bindingForRedactableBirthday(_ birthdayHeight: RedactableString) -> Binding<String> {
-        self.binding(
-            get: { _ in birthdayHeight.data },
-            send: { .birthdayInputChanged($0.redacted) }
-        )
-    }
-    
     func validateMnemonic() -> NighthawkTextEditor.ValidationState {
         self.isValidMnemonic ? .valid : .invalid(error: L10n.Nighthawk.ImportWallet.invalidMnemonic)
     }
     
     func validateBirthday() -> NighthawkTextFieldValidationState {
-        (self.birthdayHeight.data.isEmpty ||
-        (!self.birthdayHeight.data.isEmpty && self.birthdayHeightValue != nil))
+        (self.birthdayHeight.isEmpty ||
```

Comment:
Consider adding tests that track changes in the code that break the expectations of these validations. It will help avoiding form errors and time of manual testing

Reply:
Yes, automated testing in on my to-do list. I will start revamping the test suite in coming PRs